main/export-options.js

```diff
@@ -46,17 +46,21 @@ const getExportOptions = () => {
       continue;
     }
 
-    const plugin = require(json.pluginPath);
-
-    for (const service of plugin.shareServices) {
-      for (const format of service.formats) {
-        options.find(option => option.format === format).plugins.push({
-          title: service.title,
-          pluginName: json.name,
-          pluginPath: json.pluginPath,
-          apps: json.name === '_openWith' ? apps.get(format) : undefined
-        });
+    try {
+      const plugin = require(json.pluginPath);
+
+      for (const service of plugin.shareServices) {
+        for (const format of service.formats) {
+          options.find(option => option.format === format).plugins.push({
+            title: service.title,
+            pluginName: json.name,
+            pluginPath: json.pluginPath,
+            apps: json.name === '_openWith' ? apps.get(format) : undefined
+          });
+        }
       }
+    } catch (error) {
+      console.log(error);
```

Comment:
Can you log this to Sentry? So we can catch it and it could help us potentially fix it.